src/sqlfluff/core/rules/std/L006.py

```diff
@@ -1,6 +1,8 @@
 """Implementation of Rule L006."""
 
 
+from typing import Tuple, List
```

Comment:
I don't see any corresponding changes to the test file, `L006.yml`. Should there be new or updated tests to go with this?

Reply:
All the existing tests are still totally relevant. I take your point on maybe adding some extra tests though.